library/src/webapp/editor/ckextraplugins/oxpoints/dialogs/oxpoints.js

```diff
@@ -192,7 +193,7 @@ CKEDITOR.dialog.add('oxpointsDialog', function(editor) {
       // embed assets into the node
       embedAssetsInCKEditorNode({
         node: node,
-        js: [path + 'js/init-oxpoint-map.js', 'http://maps.google.com/maps/api/js?sensor=false&callback=initOxPointMap'],
+        js: [path + 'js/init-oxpoint-map.js', 'https://maps.google.com/maps/api/js?sensor=false&callback=initOxPointMap'],
```

Comment:
I think you can normally just use protocol relative URLs eg:
`://maps.google.com/....`